templates/puppetboard-ingress.yaml

```diff
@@ -25,6 +27,9 @@ spec:
         paths:
           - path: /{{ rest $url | join "/" }}
             pathType: Prefix
```

Comment:
This seems odd if we're guarding pathType right after.

Reply:
This is a leftover. Thanks.

Reply:
Removed it.